articles/errors/libraries/auth0-js/invalid-token.md

```diff
@@ -0,0 +1,23 @@
+---
+public: false
+---
+# HS256-Signed Tokens
+
+Error: calls made to the **/userInfo** return the following:
+
+```
+accessToken parameter is not valid
+```
+
+## Cause
+
+Calls made to the **/userInfo** endpoint using Access Tokens validated using the HS256 algorithm cannot be validated.
+
+## Affected Versions
+
+* auth0.js version 9
+* Lock version 11
+
+## Troubleshooting
+
+Change the value of your **responseType** parameter to **token id_token**. This will allow you to use an Access Token with the **/userInfo** endpoint.
```

Comment:
- Change the value of your **responseType** parameter to **token id_token**, so you get an access_token in the response
- Change your client's hash algorithm to RS256. Go to https://manage.auth0.com/#/clients, select your client, click "Show advanced settings", click on OAuth and change the value of "JsonWebToken Signature Algorithm" to RS256